src/sage/combinat/integer_vectors_mod_permgroup.py

```diff
@@ -763,18 +788,192 @@ def __iter__(self):
             [3, 1, 1, 2]
             [3, 0, 2, 2]
             [2, 2, 2, 1]
+
+            Check that :issue:`36681` is fixed::
+            sage: I = IntegerVectorsModPermutationGroup(
+            ....:     PermutationGroup([], domain=[]), sum=0)
+            sage: list(iter(I))     # Single empty vector
```

Comment:
Personally, I'd remove the two comments, here and two lines below, because they say exactly what the output is saying anyway.